curated_transformers/pipe.py

```diff
@@ -28,8 +28,10 @@
     assigns=["doc._.trf_data"],
     default_config={"model": DEFAULT_TRANSFORMER_MODEL},
 )
-def make_transformer(nlp: Language, name: str, model: Model) -> "Transformer":
-    return Transformer(nlp.vocab, model, name)
+def make_transformer(
+    nlp: Language, name: str, model: Model, frozen: bool = False
```

Comment:
I wonder if we should

```suggestion
    nlp: Language, name: str, model: Model, *, frozen: bool = False
```

to make sure that if someone uses this factory method directly, they do not rely on positional arguments.